internal/jujucommands/jujucommands.go

```diff
@@ -13,21 +13,46 @@ import (
 	"github.com/mitchellh/go-linereader"
 )
 
-type outputLine struct {
+var (
+	cmdPrefix = "juju"
+)
```

Comment:
Since the command runner has a prefix field, you can delete this and then you won't need to do patching from tests.